.packit.yml

```diff
@@ -12,12 +12,15 @@ jobs:
     trigger: pull_request
     metadata:
       targets:
-        - fedora-development
+        - fedora-34
+        - fedora-rawhide
```

Comment:
Why not to use `fedora-development` on both places?

Reply:
I wanted it to be explicit since this is for the f34-devel branch, I can change both to -development

we actually had a brief discussion in our team if eln should be part of -development alias

please let me know what you prefer :)

Reply:
I don't think we want to have ELN in `-development` because it's not supported by TFT so it can't be used in tests.

And when I'm thinking about it lets be explicit here. I have to remove the `34` line on master so or so. So it will be easier to remove one line instead of changing it :).

Reply:
agreed, that would be my argument that ELN should not be in the -development alias; also many projects wouldn't even build in there.

Please let me know if I should change something in here or it's good as it is.